docs/specifications/4-standard-compliance-suite.md

```diff
@@ -0,0 +1,166 @@
+# Standard Compliance Suite
```

Comment:
I would love for us to start by defining the requirements of a the compliance suite.

What is it? What is it going to be used for, by who and what information must it carry?